### PR TITLE
ENDOC-47 Add install documentation for Google Kubernetes Engine

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
This branch is currently deployed to Staging  - https://es-entando.github.io/next/tutorials/devops/installation/google-cloud-platform/

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+      - For example: `ENTANDO_DEFAULT_ROUTING_SUFFIX: 35.223.161.214.nip.io`
+      - We’re using <https://nip.io> because we need wildcard dns address resolution however nip.io is not required. If your enterprise has a different internal dns resolution scheme for development instances you can use that or other alternative dns services like xip.io.
+   - If not already present, set these values to utilize nginx as the ingress controller and file system groups for persistent volume access:
+      - `ENTANDO_REQUIRES_FILESYSTEM_GROUP_OVERRIDE: "true"`
+      - `ENTANDO_INGRESS_CLASS: "nginx"`
+   - You can also disable the operator limit checking with this setting:
+      - `ENTANDO_K8S_OPERATOR_IMPOSE_DEFAULT_LIMITS: "false"`
+   - See [Appendix B](#appendix-b-example-values-yaml-file-for-helm-quickstart) for an example values.yaml 
+3. Create the Entando namespace: `kubectl create namespace entando`
+4. Update helm dependencies: `helm dependency update`
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md:159]: @joewhite101 , @w-caffiero-entando , this uses Helm 3 which comes pre-installed for Google Cloud Shell. I didn't have any issues but is that a concern?

Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md:159]: Pretty sure the helm quickstart won't work with the helm 3 client

Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md:159]: @joewhite101 Entando started up fine and I ran through the steps with Google Cloud Shell twice. The command line arguments were slightly different but I documented what worked for me.  I just double checked the version and it's definitely Helm 3.
`n_shaw@cs-6000-devshell-vm-2b5c256d-2302-420e-8133-d10cc25b203c:~$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
n_shaw@cs-6000-devshell-vm-2b5c256d-2302-420e-8133-d10cc25b203c:~$ ls
entando-helm-quickstart-6.2.0-sprint4-rc  entando-k8s-custom-model-6.1.5  hello  ingress-resource.yaml  README-cloudshell.txt`

Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md:159]: Discussed with @joewhite101 , moving ahead with this version.

**Pull request merged**